snapshotTestContent/DuplicateGCRoutes/base_snapshot/15-bARB1JbcuAI0vHc6fsh_xQcLO

```diff
@@ -0,0 +1 @@
+{"lastEditDetailsKey":{"user":{"id":"tylercai@microsoft.com","name":"Tyler Cai","email":"tylercai@microsoft.com","oid":"030a20b7-8d91-44a7-90b9-0b67dc0b94b8"},"timestamp":1651091104028}}
```

Comment:
You need to scrub out your name, id, address. Basically all PIIs.
We need to update the README in tests/snapshot to add this step.

Reply:
It just says 
> Run the tests to validate that all the tests pass with the new snapshots.
So I ran `npm run test` in the FF/packages/test/snapshots repo and that passed.